RFC-676-Contributor-Council-Charter.md

```diff
@@ -0,0 +1,222 @@
+# CDK Contributor Council Charter (RFC) — November, 2024
+
+## Purpose of the Council
+
+To improve visibility into CDK, increase feedback opportunities, and build a stronger, more inclusive community, the AWS CDK team proposes forming a
+Contributor Council.
+
+*Original Author(s)*: [@haubles](https://github.com/haubles), [@billfine](https://github.com/billfine)
+
+*Tracking Issue*: https://github.com/aws/aws-cdk-rfcs/issues/676
+
+## Principles guiding this RFC
+
+1. **Maintain AWS supportability.** In order for AWS to provide on-call resources tasked with resolving high-impact bugs or issues in CDK for
+customers and the community, AWS must ensure that its standards for processes—for example, security reviews—are met. Therefore, AWS controls
+the CDK release process.
+2. **Increase community influence**. The community’s input and active engagement is essential for the long-term health and success of the CDK
+project. Therefore, AWS is committed to creating more robust and clearly-defined channels for soliciting community feedback, and acting on that
+feedback. We are committed to increasing opportunities and building new processes for accepting more community contributions to the codebase.
+3. **Provide visibility into AWS-led initiatives.** We will provide the CDK’s community with visibility into our roadmap and AWS-led contributions.
+We will give the community a voice and a hand in our decision-making processes. We must communicate our roadmaps, plans, and decisions openly and
+promptly, and deliver on them consistently, for the long-term success of CDK.
+
```

Comment:
When scanning the document by section titles, beginning with Maintain AWS supportability can mislead readers about the project’s intent by suggesting AWS is distancing itself from the CDK. To emphasize community collaboration up front, move this principle to the end of the list and rename it Maintain AWS support. This way, the document clearly signals AWS’s continued involvement while prioritizing the community’s influence and visibility.